elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -426,6 +422,10 @@ class PfeContentSet extends PFElement {
       if (sets) view.appendChild(sets);
     }
 
+    // Render or re-cascade properties to the component after update
+    if (!this._rendered) this.render();
+    else this.cascadeProperties(this.viewAll);
+
```

Comment:
So if it's being rendered for the first time it will take care of cascading properties, right?

Reply:
Ok I see the cascade is in pfelement's render().  nice!